docs/wiki/CodeTour.md

```diff
@@ -0,0 +1,21 @@
+<!-- markdownlint-disable -->
+## Contents
+<!-- markdownlint-restore -->
+
+- [Contents](#contents)
+- [E2E Testing pipeline tour](#e2e-testing-pipeline-tour)
+
+---
+
+As par of the ongoing development of ALZ Bicep, we are beginning to add [CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) tours of the code. Tours can be found in [tours_repo]. 
+
```

Comment:
Can we add a link and a small blurb about how to use a codetour?

Like https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour&ssr=false#user-content-opening-tours ?